diff --git a/node_modules/element-plus/es/components/table/src/table-column/index.mjs b/node_modules/element-plus/es/components/table/src/table-column/index.mjs
index fb2cf9e..2533e9e 100644
--- a/node_modules/element-plus/es/components/table/src/table-column/index.mjs
+++ b/node_modules/element-plus/es/components/table/src/table-column/index.mjs
@@ -41,7 +41,7 @@ var ElTableColumn = defineComponent({
       updateColumnOrder
     } = useRender(props, slots, owner);
     const parent = columnOrTableParent.value;
-    columnId.value = `${parent.tableId || parent.columnId}_column_${columnIdSeed++}`;
+    columnId.value = `${parent.tableId || parent.columnId}_column ${parent.tableId || parent.columnId}_column_${props.prop || props.property || props.label || props.type || 'default'}`;
     onBeforeMount(() => {
       isSubColumn.value = owner.value !== parent;
       const type = props.type || "default";
